src/containers/Balance/Balance.tsx

```diff
@@ -91,10 +89,11 @@ class Balance extends React.Component<Props, State> {
 
   private onEnterTransferTokens(event: ChangeEvent<HTMLInputElement>): void {
     const { transfer } = this.state
-    const { value: amount } = event.target
+    const { value: inputValue, validity } = event.target
+    const amount = validity.valid ? inputValue : transfer.amount
     const myBalance = this.getMyBalance()
 
-    if (!myBalance || amount.includes('.')) {
+    if (!myBalance) {
```

Comment:
To be honest, I don't like this check. It feels a bit of a copout.

If there was a better way I would suggest it.

Reply:
The way polkadot handles it is not stopping people from adding more numbers but has a red bar.

![Screenshot 2020-10-01 at 11 40 43](https://user-images.githubusercontent.com/48058984/94793566-f26e4980-03da-11eb-86ad-58d85bc648fc.png)

Maybe, this would be a better visual queue for the users.